### PR TITLE
🎨 adjustments

### PR DESCRIPTION
**Pull request by vojtaholik**
I've noticed a few things while working on #99 and figured I'd suggest them here:

- Consistent "home" navigation
- Nav links & code blocks style adjustments
- Disable css smooth scroll
- Favicon update

### CODE REVIEW

File: src/templates/post.js [hunk 1]
```diff
@@ -34,8 +33,6 @@ export default function Post({data: {site, mdx}}) {
     <Layout
       site={site}
       frontmatter={mdx.fields}
-      pageTitle={`Kent C. Dodds ${isWriting ? 'Writing ' : ''}Blog`}
-      headerLink={isWriting ? '/writing/blog' : '/blog'}
```
Comment (kentcdodds) [inline src/templates/post.js]: I think this breaks the https://kentcdodds.com/writing site.

Comment (vojtaholik) [reply src/templates/post.js]: doesn't seem to? I checked locally as well as [here](https://deploy-preview-103--kentcdodds.netlify.com/writing/blog/write-the-story-that-works) and [here](https://deploy-preview-103--kentcdodds.netlify.com/writing/).

Comment (kentcdodds) [reply src/templates/post.js]: Clicking on "Kent C. Dodds" on this page should go to `/writing` but it goes to `/`:

https://deploy-preview-103--kentcdodds.netlify.com/writing/blog/write-the-story-that-works

Clicking on "Blog" on this page should go to `/writing/blog` but it goes to `/blog`:

https://deploy-preview-103--kentcdodds.netlify.com/writing/

I think I need to rework a few things (maybe even from a design aspect) to make a more clean separation between the main side of my site and the novelist side of my site.

Comment (kentcdodds) [reply src/templates/post.js]: I _would_ love all the other changes though. I think they're super!

File: src/components/header.js [hunk 2]
```diff
@@ -104,28 +122,28 @@ const Header = ({
           <MobileNav color={headerColor} />
           <NavLink
             headerColor={headerColor}
-            to="/blog"
+            to="/blog/"
```
Comment (kentcdodds) [inline src/components/header.js:125]: Why add the `/` to these?

**Review by kentcdodds — COMMENTED** [review]
I think this is good. Just one question.

File: src/components/header.js (hunk 2 above)
Comment (vojtaholik) [reply src/components/header.js:125]: I found out that using `/` sets Link's `activeClassName` on initial page load which wasn't the case before. https://cl.ly/4f495cb95e2f 

So if you visit [this](https://deploy-preview-103--kentcdodds.netlify.com/talks/) page, `Talks` link is highlighted whereas visiting [this](https://kentcdodds.com/talks/) page it's not.

Comment (kentcdodds) [reply src/components/header.js:125]: Makes sense. Thanks!

**Pull request merged**